src/NuGet.Core/NuGet.ProjectModel/ProjectLockFile/PackagesLockFileUtilities.cs

```diff
@@ -380,8 +461,15 @@ private static bool HasP2PDependencyChanged(IEnumerable<LibraryDependency> newDe
 
                 if (matchedP2PLibrary == null || !EqualityUtility.EqualsWithNullCheck(matchedP2PLibrary.VersionRange, dependency.LibraryRange.VersionRange))
                 {
+                    // Do it in a similar way as the above.
```

Comment:
I think the `do it in a similar way as above` part refers to actually detecting the differences, rather than just saying it has changed.

Reply:
Yes Nikolche, `HasP2PDependencyChanged` method returns `true` if `P2P2P` dependencies have changed,